CONTRIBUTING.md

```diff
@@ -241,6 +241,32 @@ Good docs make developers happy, and we love happy developers! We've got a few d
 * Tutorials/examples
 * Docstrings in Python functions in RST format - generated by Sphinx
 
+### Documentation guidelines
+
+1. Decide if your page is a **guide or a tutorial**. Make sure it fits its section.
+2. Use “**you**” instead of “we” or “I”. It **engages** the reader more.
+3. **Sentence case** for headers. (Use [https://convertcase.net/](https://convertcase.net/) to check)
+4. Keep sentences short. If possible, **fewer than 13 words**.
+5. Only use `backticks` for direct references to code elements.
+6. Jina product names should be capitalized and not backticked (Flow, Executor, Hub etc.).
```

Comment:
In our [blogging guide](https://medium.com/@jina_ai/jina-ai-blogging-style-guide-69fbce4d0c6c) I've got a list at the end with how to capitalize things. Since some words like DocArray, JCloud don't just have capital first letter